extensions/README.md

```diff
@@ -0,0 +1,42 @@
+# Microsoft Authentication Extensions for Node
+The Microsoft Authentication Extensions for Node offers secure mechanisms for client applications to perform cross-platform token cache serialization and persistence. It gives additional support to the Microsoft Authentication Library for Node (MSAL).
+
+MSAL Node supports an in-memory cache by default and provides the ICacheManager to perform cache serialization. Developers are required to implement their own cache persistance across multiple platforms and Microsoft Authentication Extensions makes this simpler.
+
+The supported platforms are Windows, Mac and Linux.
```

Comment:
We should consider being specific about which versions of Linux we have tested.

Reply:
Agreed. Added item to backlog to test on multiple distributions and add to README